<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>总览</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>631</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="plkh">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>批量开户</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plfk">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>批量发卡</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>清除数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>60</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="time1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>时间未设置 请设置时间</string>
      </property>
     </widget>
     <widget class="QPushButton" name="plkcz">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>批量卡操作</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>查询</string>
     </attribute>
     <widget class="QLineEdit" name="xuehao">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="xingming">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="ask">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>591</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fk">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>40</y>
        <width>92</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>发卡</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bk">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>40</y>
        <width>92</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>补卡</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gs">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>80</y>
        <width>92</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>挂失</string>
      </property>
     </widget>
     <widget class="QPushButton" name="jg">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>80</y>
        <width>92</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>解挂</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cz">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>320</y>
        <width>92</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>充值</string>
      </property>
     </widget>
     <widget class="QPushButton" name="kh">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>开户</string>
      </property>
     </widget>
     <widget class="QPushButton" name="xh">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>120</y>
        <width>92</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>销户</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="money">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>280</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>260</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>输入充值金额</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="time2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>时间未设置 请设置时间</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>360</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>输入起始日期</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="starttime">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>380</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>420</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>输入截至日期</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="endtime">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>440</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ask_xf">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>480</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>查询消费记录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changepw">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>120</y>
        <width>92</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>修改密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hz">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>160</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>汇总窗口数据</string>
      </property>
     </widget>
     <widget class="QPushButton" name="wj">
      <property name="geometry">
       <rect>
        <x>651</x>
        <y>207</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>挖掘</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>时间</string>
     </attribute>
     <widget class="QLineEdit" name="year">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>125</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>年</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="month">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>月</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="day">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>124</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>日</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hour">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>时</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="min">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>124</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>分</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sec">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>124</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>秒</string>
      </property>
     </widget>
     <widget class="QPushButton" name="settime">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>280</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="msec">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>120</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>10毫秒</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>消费</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>窗口：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="window">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>180</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>卡号：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cardid">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>金额：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="payment">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>250</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pay">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="xjzh">
   <property name="text">
    <string>新建账户</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>删除账户</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>挂失</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>解挂</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>补卡</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>开卡</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>充值</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>查询余额</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>查找账户</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>食堂数据</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
